indico/web/forms/fields/principals.py

```diff
@@ -66,19 +70,24 @@ def __init__(self, *args, **kwargs):
         self.allow_emails = kwargs.pop('allow_emails', False)
         self.groups = kwargs.pop('groups', False)
         self.allow_networks = kwargs.pop('allow_networks', False)
+        self.allow_registration_form = kwargs.pop('allow_registration_form', False)
```

Comment:
`allow_registration_forms` (make sure to also update the place where this field is used)